src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Please see if this is required.

Reply:
yes this the create react app recognizes path through index.js